packages/editor/src/client/global.less

```diff
@@ -93,3 +93,16 @@
     border-color: #1675e0;
   }
 }
+
+
+.authorLinks {
```

Comment:
`.authorLinks > div` will only target direct div children of .authorLinks (so it would not work if the div is a child of a paragraph for example), while the current selector `.authorLinks div div div div` will target any descendant div that is any descendant div of any descendant div of  any descendant div of .authorLinks.

for example:

```
// .authorLinks div div div div

.authorLinks
  p
     div
       p
          div
              p
                div
                   p
                     div <--- would have overflow: visible
                        div <--- would have overflow: visible
                           p
                              div <--- would have overflow: visible
                      

// .authorLinks > div
.authorLinks
  p
    div <--- would not work
  div  <--- would have overflow: visible
```